packages/wallets/src/components/index.ts

```diff
@@ -0,0 +1,8 @@
+export * from './DesktopWalletsHeader';
+export * from './WalletBalance';
+export * from './WalletCardIcon';
+export * from './WalletCurrencyTitle';
+export * from './WalletLandingCompanyBadge';
```

Comment:
```suggestion
export * from './WalletListCardBadge';
```
@thisyahlen-deriv This one maybe can be generic if we're going use it in other places, So we can name it something not tied with `WalletListCard` 🤔